.github/workflows/release.yml

```diff
@@ -35,4 +35,22 @@ jobs:
           version: latest
           args: release --clean
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  helm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      
+      - name: Registry login
+        run: helm registry login ghcr.io -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Helm lint
+        run: helm lint chart/.
+      
+      - name: Helm package
+        run: helm package chart/. -d chart/.
```

Comment:
is the `chart./` a typo or is this correct?

Reply:
dot was unnecessary, rm it